src/css/main.scss

```diff
@@ -123,6 +123,7 @@ body {
 }
 
 .form-group {
+  font-size: .885rem;
```

Comment:
Please check if this does not affect other field components too (LabelField and BaseField in case if there is fieldPreview == true)